src/module/profile/domain/profile.entity.ts

```diff
@@ -0,0 +1,22 @@
+import { Entity, PrimaryGeneratedColumn, Column } from 'typeorm';
+
+@Entity()
+export class Profile {
```

Comment:
All Entities should extend from the Base entity defined in the domain folder of the common module.
That base entity contains the id, updateAt, and createdAt properties.